product_brand_print/reports/sale_reports.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<odoo>
+    <template id="report_saleorder" inherit_id="sale.report_saleorder_document">
+                <xpath expr="//table/thead//tr[1]//th[1]" position="after">
+            <th class="text-left">Brand</th>
+        </xpath>
+        <xpath
+            expr="//table/tbody[hasclass('sale_tbody')]//tr[1]//td[1]"
```

Comment:
avoid using indexes as this may leads to different behavior if the base report change.

```suggestion
            expr="//td[@name='td_name']"
```